lib/themes-panel.coffee

```diff
@@ -147,6 +153,35 @@ class ThemesPanel extends View
     @activeUiTheme = @getActiveUiTheme()
     @activeSyntaxTheme = @getActiveSyntaxTheme()
     @populateThemeMenus()
+    @toggleCurrentThemeButtons()
+    @handleCurrentThemeButtons()
+
+  handleCurrentThemeButtons: ->
+    @currentUiThemeSettings.on 'click', (event) =>
+      event.stopPropagation()
+      @parents('.settings-view').view()?.showPanel(@activeUiTheme, {
+        back: 'Themes',
+        pack: atom.themes.getActiveThemes().filter((theme) -> theme.metadata.theme is 'ui')[0].metadata
+      })
+
+    @currentSyntaxThemeSettings.on 'click', (event) =>
+      event.stopPropagation()
+      @parents('.settings-view').view()?.showPanel(@activeSyntaxTheme, {
+        back: 'Themes',
+        pack: atom.themes.getActiveThemes().filter((theme) -> theme.metadata.theme is 'ui')[0].metadata
+      })
+
+  toggleCurrentThemeButtons: ->
+    if atom.config.get(@activeUiTheme)?
```

Comment:
`atom.config.get()` could return any empty `{}` in which case I'd say the button shouldn't be shown.